### ISSUE TITLE
Marked as proprietary while under GPL

### ISSUE THREAD

**Issue by mokkin** [Zettlr/Zettlr#540]
While installing Zettlr with the help of Ubuntu Software the application is marked as proprietary:
![grafik](https://user-images.githubusercontent.com/2938748/74384013-a7bea380-4df0-11ea-9a19-35ab467ebf09.png)
I'm not sure if this is by default because of installing a deb-Package from outside the repos or there is a false setting.

**Comment by nathanlesage** [comment]
I have no idea how Debian decides upon which license to use. Please feel free to look that up, because I have no idea which field it uses to mark something as proprietary

**Comment by alerque** [comment]
The `debian/copyright` file has the meta data about licenses, including a machine readable `License:` field that needs to be recognized as a known license plus a long form human readable copy of the actual license.

**Comment by nathanlesage** [comment]
Thanks for the info! :)

I just had a look at the Electron-Builder config, and it appears that Linux targets draw their license from the `license`-field in the `package.json`, but this is already a machine-readable GPL v3-string, so I don't know what's going on: https://www.electron.build/configuration/configuration#overridable-per-platform-options

In this documentation, I can see the license being expressed as `GPL-3+`, which I haven't seen before ... but Electron builder should re-cast `GPL-3.0` to `GPL-3+`, if that's the required format for Debian, because the former one is the official and correct license for a `package.json`

**Comment by marado** [comment]
I have no experience with Electron-Builder, but the correct SPDX format for this project is GPL-3.0-only, as explained in the description of #672 .

**Comment by nathanlesage** [comment]
Does this then fix the Debian installer license field?

**Comment by marado** [comment]
No, it seems that this is an electron-builder bug:
https://github.com/electron-userland/electron-builder/issues/4270

**Comment by marado** [comment]
This issue still exists.

**Comment by nathanlesage** [comment]
@marado You have indicated that it apparently is a bug in electron-builder, right? If so, then I don't see any reason why we should keep it open here. After all, it's a minor nuisance and not a real issue for Zettlr, and furthermore, it'll be fixed as soon as Electron Builder upgrades, am I right? Or is any further action from us required?

**Comment by marado** [comment]
Yes, it is electron's issue https://github.com/electron-userland/electron-builder/issues/4270 (still open).
Without knowing how will it be solved, we can't be sure Zettlr won't need to do anything on its side, once the issue is solved there.
